src/nidigital/system_tests/test_system_nidigital.py

```diff
@@ -107,5 +108,69 @@ def configure_session(session):
         initial_state_low_pins='',
         initial_state_tristate_pins='')
 
-    session.load_pattern(os.path.join(test_files_dir, 'pattern.digipat'))
+
+def get_test_file_path(test_name, file_name):
+    return os.path.join(test_files_base_dir, test_name, file_name)
+
+
+def test_fetch_capture_waveform(multi_instrument_session):
+    if sys.version_info.major >= 3:
```

Comment:
Worth commenting why we limit this to Python 3.

Reply:
Done